app/views/projects/show.html.erb

```diff
@@ -38,12 +38,14 @@
     </div>
   </div>
   <div class='col-md-4'>
-    <ul>
+    <table class='table table-striped'>
       <% @project.summary.each do |key, value| %>
-        <li>
-          <%= key.to_s.humanize %>: <strong><%= value %></strong>
-        </li>
+        <tr>
+          <td>
+            <%= key.to_s.humanize %>: <strong><%= value %></strong>
```

Comment:
personally I'd split this in to two columns since it's key -> value in nature.

i.e.

```
<tr>
  <td><%= key.to_s.humanize %></td>
  <td><%= value %></td>
</tr>
```